config/software/pywin32.rb

```diff
@@ -1,23 +0,0 @@
-name "pywin32"
-default_version "219"
-
-dependency "python"
-dependency "pip"
-
-build do
-  relative_path "pywin32-#{version}"
-  # Switch on the architecture
-  pip "install pypiwin32==#{version} "\
-      "--prefix=#{windows_safe_path(install_dir)}/embedded"
-  # pywintypes is patched, it doesn't work on Python > 2.7 otherwise
-  # Since we don't install it from source, we can't use the patch option of omnibus
-  # Here is a manual patch, using the same options as omnibus
```

Comment:
Do we not need the patch on `pywintypes` anymore?

Reply:
I don't think so since https://github.com/mhammond/pywin32/commit/45975cea95b13a0e519e21866ffa8748b853899c#diff-ee83992a2b5b37fd93d57e1033679cdc

Reply:
makes sense thanks 👍 can you also remove the patch file?